### DISCUSSION TITLE
VSync option doesn't limit framerate

### DISCUSSION THREAD

**Discussion by JGSVb** [ppy/osu#13341] [Q&A]
VSync option isn't working as I think it should: It doesn't limit drawing at 60fps, and osu runs at 120-200fps. But when I leave the input focus out of the osu window, the draw framerate changes to 60fps. I searched in the source if there was any way to set custom framerate in the config file, but I couldn't find. Maybe I'll need to compile osu-framework to make it run at 60fps.

I can provide more details if needed.

![input focus on osu window](https://user-images.githubusercontent.com/59124659/120899543-2368e300-c628-11eb-9939-e1e8bae7bae4.png)
![input focus out of osu window](https://user-images.githubusercontent.com/59124659/120899565-3a0f3a00-c628-11eb-8f48-1756817bccb3.png)

Operating System: Linux Mint 20
Architecture: x86_64
Kernel: 5.4.0-73-generic

2021.602.0

**Comment by peppy** [comment]
This is more than likely going to be an issue with your graphics drivers.

**Comment by smoogipoo** [comment]
What's the output of running `xrandr`, or the Wayland equivalent if you use Wayland?
